modules/homepage/src/main/frontend/src/themePage/Sidebar/sidebarStyle.jsx

```diff
@@ -88,17 +87,18 @@ import {
       }
     },
     logoImage: {
-      width: "30px",
+      width: "100%",
       display: "inline-block",
       maxHeight: "30px",
-      marginLeft: "10px",
-      marginRight: "15px"
     },
     img: {
-      width: "32px",
+      maxWidth: "90%",
```

Comment:
This gives the logo some space left and right after we relax the `maxHeight`.
```suggestion
      maxWidth: "70%",
```